api/src/main/java/io/strimzi/api/kafka/model/kafka/listener/GenericKafkaListenerConfigurationBootstrap.java

```diff
@@ -114,6 +115,19 @@ public void setLoadBalancerIP(String loadBalancerIP) {
         this.loadBalancerIP = loadBalancerIP;
     }
 
+    @Description("External IPs associated to the nodeport service. " + 
+            "These IPs are used by clients external to the cluster to access the kafka brokers. " +
```

Comment:
I guess "cluster" is referring to Kubernetes cluster not Kafka one. It's better to clarify.